weblate/trans/filter.py

```diff
@@ -128,3 +128,32 @@ def get_filter_choice(project=None):
             for label in project.label_set.values_list("name", flat=True)
         )
     return result
+
+
+def get_search_query_choices():
```

Comment:
This should reuse FilterRegistry otherwise we're again duplicating the code (`get_filter_choice` should be rewritten at some point as well).